examples/story-book.gpt

```diff
@@ -0,0 +1,64 @@
+tools: story-writer, story-illustrator, server, mkdir, sys.write, sys.read, sys.download
+description: Writes a children's book and generates illustrations for it.
+args: story: The story to write and illustrate. Can be a prompt or a complete story.
+
+Do the following steps sequentially:
+
+1. Create a folder called story with mkdir.
+2. If ${story} is a prompt and not a complete children's story, call story-writer
+   to write a story based on the prompt.
+3. Take ${story} and break it up into logical chunks of text.
+4. For each chunk:
+   - Call story-illustrator to illustrate it.
+   - Download the illustration to a file at story/<chunk_number>.png.
+5. Compile the chunks and their illustrations into an HTML file to story/story.html where
+   the text and illustrations are side-by-side scrolling down with good padding. Make sure
+   the image is not too large and the text is not too small.
+6. Finally, call serve to serve the story as a webpage and print "Serving at http://localhost:8081".
+
+---
+name: story-writer
+description: Writes a story for children
+args: prompt: The prompt to use for the story
+
+Write a story with a tone for children based on ${prompt}.
+
+---
+name: story-illustrator
+temperature: 0.5
+# assumes you have image-generation.gpt installed
+tools: ../../image-generation/tool.gpt
```

Comment:
This makes an assumption that `image-generation` is cloned to the same directory as `gptscript` is on the user machine. Do we call that out anywhere as something the user needs to do in order to run this?

Reply:
There is now a README.md for this example that includes an installation step.